include.mk

```diff
@@ -0,0 +1,4 @@
+SETUP_GOTOOLS := @echo "Installing golint and vet" && go get -u github.com/golang/lint/golint && go get -u -v golang.org/x/tools/cmd/vet
+GO_VET := @echo "Running go vet ..." && go vet ./...
+GO_LINT := @echo "Running golint ..." && golint ./...
+GO_TEST := @echo "Running tests ..." && go test -v ./...
```

Comment:
If you move the make targets [test, test-unit, lint, vet, setup-gotools] into this file, you'll probably no longer have a need to create make targets inside of make variables.